object_storage_api/schemas/image.py

```diff
@@ -28,3 +28,11 @@ class ImageSchema(CreatedModifiedSchemaMixin, ImagePostMetadataSchema):
     file_name: str = Field(description="File name of the image")
     primary: bool = Field(description="Whether the image is the primary for its related entity")
     thumbnail_base64: str = Field(description="Thumbnail of the image as a base64 encoded byte string")
+
+
+class ImageGetUrlInfoSchema(ImageSchema):
```

Comment:
I think given the way we have named things in IMS `ImageGetUrlInfoSchema` would imply it is the UrlInfo object inside the ImageGetSchema. I think the previous ImageSchema should be renamed ImageMetadataSchema, so that this can be the true ImageGet.